db/re/item_db_etc.yml

```diff
@@ -41534,6 +41534,20 @@ Body:
       DropEffect: CLIENT
     Script: |
       bonus bDelayrate,-4;
+  - Id: 27344
+    AegisName: Witch_Zilant_Card
+    Name: Witch Zilant Card
+    Type: Card
+    Buy: 20
+    Weight: 10
+    Locations:
+      Garment: true
+    Flags:
+      DropEffect: CLIENT
+    Script: |
+      bonus2 bMagicAtkEle,Ele_Fire,100;
+      bonus2 bSubEle,Ele_Fire,50;
+      /*TODO Imprison State Resist*/
```

Comment:
I didn't test yet, maybe
```
bonus2 bResEff,SC_WHITEIMPRISON,100;
```
could work?
If it works please export the constant
```
export_constant2("Eff_WhiteImprison",SC_WHITEIMPRISON);
```

Reply:
It works

Thanks!